tasks/sprite.js

```diff
@@ -0,0 +1,28 @@
+const gulp = require('gulp');
+const config = require('./../config');
+
+gulp.task('convertIconsToSprites', function() {
+	const runSequence = require('run-sequence');
+
+	runSequence('compileSvg', 'copySvgFile');
+})
+
+gulp.task('compileSvg', function () {
+	const svgSprite = require('gulp-svg-sprites');
+
+	return gulp.src(config.global.src + config.global.resources + '/icons/*.svg')
+		.pipe(svgSprite({
+			baseSize: 20,
+			cssFile: '../../scss/fonts/_sprite.scss',
+			svg: {
+				sprite: "svg/sprite.svg"
+			},
+			preview: false
+		}))
+		.pipe(gulp.dest(config.global.src + config.global.resources + '/icons/sprite'));
+});
+
+gulp.task('copySvgFile', function() {
+	return gulp.src(config.global.src + config.global.resources + '/icons/sprite/svg/sprite.svg')
+		.pipe(gulp.dest(config.global.dev + config.global.resources + '/svg'));
```

Comment:
Why do wen need another copy task? This should be done all at once inside the compileSvg task